blog/articles/views/create_article.py

```diff
@@ -0,0 +1,18 @@
+from django.http import HttpResponse, HttpRequest
+from django.shortcuts import render
+from django.views import View
+from django.shortcuts import redirect
+from articles.models import Article
+
+
+class CreateArticle(View):
+    def get(self, request: HttpRequest) -> HttpResponse:
+        return render(request, "create_article.html")
+
+    def post(self, request: HttpRequest) -> HttpResponse:
+        title = request.POST.get("title")
+        text = request.POST.get("text")
+        article = Article(title=title, text=text)
```

Comment:
Looks like I can get 500 err. For example what will happen if I send empty title or text?